lms/static/sass/_developer.scss

```diff
@@ -232,3 +232,82 @@
     }
   }
 }
+
+//efischer; scratch space to make things up and see what happens
+.wrapper-msg {
```

Comment:
Please nest / scope all of these styles within .view-teams {} to make sure that these styles won't impact other parts of the LMS.